contrib/ptpcam/ptp.c

```diff
@@ -26,6 +26,7 @@
 #include <stdarg.h>
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
```

Comment:
Ah, thanks.  That makefile looks like our source is intended to build on Windows, although as you say probably with some binary people have to find.  It looks like cygwin can (does?) provide unistd.h, so this may build with the change.  I don't mind too much if we break it, if anyone is using it they can complain.

If we want to support Windows itself, we shouldn't use WSL; that's just supporting a different flavour of Linux.